components/admin/BearerRefresh/BearerRefresh.tsx

```diff
@@ -0,0 +1,98 @@
+import { Button } from "@components/forms";
+import Loader from "@components/globals/Loader";
+import { logMessage } from "@lib/logger";
+import { BearerResponse } from "@lib/types";
+import axios from "axios";
+import { useTranslation } from "next-i18next";
+import React, { useEffect, useState } from "react";
+
+export interface BearerRefreshProps {
+  formID: number;
+}
+
+const BearerRefresh = (props: BearerRefreshProps): React.ReactElement => {
+  const { formID } = props;
+  const [bearerTokenState, setBearerTokenState] = useState("");
+  const { t } = useTranslation("admin-templates");
+  const [errorState, setErrorState] = useState({ message: "" });
+  const [submitting, setSubmitting] = useState(false);
+
+  useEffect(() => {
+    setBearerTokenState("");
+    getBearerToken(formID);
+  }, []);
+
+  const getBearerToken = async (formID: number) => {
+    try {
+      setSubmitting(true);
+      setErrorState({ message: "" });
+      const serverResponse = await axios({
+        url: `/api/id/${formID}/bearer`,
+        method: "GET",
+        timeout: 0,
```

Comment:
Similiar to the Submit API I'm wondering if we should timeout the call after 60 seconds but we can leave infinite for the dev environment.  
[Submit API Example](https://github.com/cds-snc/platform-forms-client/blob/906a662a5630e9c7ad91080743fff2adb2f4aa05/lib/integration/helpers.ts#L270)